src/modules/ekf2/EKF/aux_global_position.cpp

```diff
@@ -103,12 +104,14 @@ void AuxGlobalPosition::update(Ekf &ekf, const estimator::imuSample &imu_delayed
 
 				if (ekf.global_origin_valid()) {
 					ekf.enableControlStatusAuxGpos();
+					_reset_counters.lat_lon = sample.lat_lon_reset_counter;
```

Comment:
Activation should actually do something, either fuse (and set AuxGPOS enabled on success) or resetHorizontalPosition in order to start (depending on the availability and health of other aid sources).

Reply:
I could add this in a followup.